sdk/storage/Swagger/Blobs/blob.json

```diff
@@ -0,0 +1,6974 @@
+{
```

Comment:
We are trying to keep all the folders directly under `sdk\<service>` as shipping packages so we probably want to consider moving these swagger changes under the corresponding package folders.

Reply:
I think this ends up being uglier, but I've split everything apart.